web/docs/guides/api.mdx

```diff
@@ -197,51 +210,86 @@ let { data: todos, error } = await supabase
 
 ```bash
 # Append /rest/v1/ to your URL, and then use the table name as the route
-curl '[SUPABASE_URL]/rest/v1/todos' \
--H "apikey: [SUPABASE_ANON_KEY]" \
--H "Authorization: Bearer SUPABASE_ANON_KEY"
+curl '<SUPABASE_URL>/rest/v1/todos' \
+-H "apikey: <SUPABASE_ANON_KEY>" \
+-H "Authorization: Bearer <SUPABASE_ANON_KEY>"
 ```
 
 </TabItem>
 </Tabs>
 
-#### Details and links:
-
-- When you create a table in Postgres, Row Level Security is disabled by default. Make sure you secure it by [enabling RLS](/docs/guides/api#securing-your-routes).
-- Never expose the `service_role` key in a browser or anywhere where a user can see it.
-- JS Reference: [select()](/docs/reference/javascript/select), 
+JS Reference: [select()](/docs/reference/javascript/select), 
 [insert()](/docs/reference/javascript/insert),
 [update()](/docs/reference/javascript/update),
 [upsert()](/docs/reference/javascript/upsert),
 [delete()](/docs/reference/javascript/delete),
 [rpc()](/docs/reference/javascript/rpc) (call Postgres functions).
 
-### Safeguards towards accidental deletes and updates
 
-For new projects, by default, the Postgres extension [safeupdate](https://github.com/eradman/pg-safeupdate) is enabled for all queries coming from the API. 
-This ensures that any `delete()` or `update()` would fail if there are no accompanying filters provided.
-This can be disabled by running the following query on the SQL editor on the dashboard:
+### GraphQL API 
 
-```sql
-alter role "authenticator"
-set "session_preload_libraries" to default;
+You can use any GraphQL client with the Supabase GraphQL API. For our GraphQL example we will use [urlql](https://formidable.com/open-source/urql/docs/). 
+
+<Tabs
+defaultValue="Javascript"
+values={[
+  {label: 'Javascript', value: 'Javascript'},
+  {label: 'cURL', value: 'cURL'},
+]}>
+<TabItem value="Javascript">
+
+```javascript
+import { createClient, useQuery } from 'urql';
+
+const client = createClient({
+  url: '<SUPABASE_URL>/grapqhl/v1',
+})
+
+const TodosQuery = `
+  query {
+    todos {
+      id
+      title
+    }
+  }
+`
+
+const [result, reexecuteQuery] = useQuery({
+  query: TodosQuery,
+})
+
+const { data, fetching, error } = result
```

Comment:
TODO: confirm this is correct

Reply:
can I pass an `apikey` header to `createClient`?

Reply:
The code used in the example app creates an Url Provider using ```  
  function getHeaders(): Record<string, string> {
    const headers: Record<string, string> = {
      apikey: process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY!,
    };
    const authorization = supabaseClient.auth.session()?.access_token;

    if (authorization) {
      headers["authorization"] = `Bearer ${authorization}`;
    }

    return headers;
  }

  const [client] = React.useState(function createUrqlClient() {
    return createClient({
      url: `${process.env.NEXT_PUBLIC_SUPABASE_URL!}/graphql/v1`,
      fetchOptions: function createFetchOptions() {
        return { headers: getHeaders() };
      },
    });
  });
```

It is important that the headers be the result of a function if fetching from auth so they get refreshed/current each time (ie, after login or out).

This is how the API Key and auth headers are set.